a1/README.md

```diff
@@ -3268,6 +3269,16 @@ Use [Gulp](http://gulpjs.com) or [Grunt](http://gruntjs.com) for creating automa
 
 **[Back to top](#table-of-contents)**
 
+## Low Level Functions
+
+  - Avoid using [Angular low-level functions](https://docs.angularjs.org/api/ng/function) such as `angular.copy()`, `angular.extend()` or even `angular.isUndefined()`.
+   
+  *Why?*: You can replace a lot Angular's low-level functions with native ES2015+ functionality such as [`Object.assign()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign) for copying objects and the [Spread Syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_operator#Copy_an_array) for copying arrays _(ie. `[...arr1]`)_.
+
+  - Note: Use a utility library like [lodash](https://lodash.com/docs/4.15.0) if you need additional functionality that is not available natively in the newest versions of JavaScript.
```

Comment:
It is better to omit `4.15.0` from the link (to always point to the latest version).